scripts/cgp_data_loader.py

```diff
@@ -39,6 +35,22 @@ def main(argv=sys.argv[1:]):
                         help='Upload bundles serially. This can be useful for debugging')
     parser.add_argument('input_json', metavar='INPUT_JSON',
                         help="Path to the standard JSON format input file")
+    parser.add_argument('-p', '--project-id', dest='project_id', default='platform-dev-178517',
+                        help='Specify the Google project ID.')
+    parser.add_argument('--aws-metadata-cred', dest='aws_metadata_cred', default=None,
+                        help='The loader by default needs no additional credentials to '
+                             'access public references, but when attempting to access '
+                             'private cloud files in order to determine size and hash '
```

Comment:
Maybe add AWS in here
```suggestion
                             'private AWS files in order to determine size and hash '
```